src/App.jsx

```diff
@@ -1,22 +1,65 @@
-import { useState } from 'react';
+import { collection, query, where, getDocs, terminate } from 'firebase/firestore';
+import { useState, useEffect } from 'react';
 import './App.css';
 import Home from './pages/Home';
+import db from '../src/utils/firebase';
```

Comment:
if you do `import {db} from '../src/utils/firebase` then you don't need to do `db.db` on the below codes.

Reply:
oh, `import {db} from '../src/utils/firebase` creates import error. change `export default { db, auth };` to `export { db, auth };` in firebase.js will fix the issue.

Reply:
That leads to a complaint from eslint:

No default export found in imported module

Reply:
yeah i got the issue and left the comment for fixing it! 😁

Reply:
I fixed it! I will open a new merge request tonight.